contracts/crowdsale/validation/UserCappedCrowdsale.sol

```diff
@@ -0,0 +1,76 @@
+pragma solidity ^ 0.4.18;
+
+import "../../math/SafeMath.sol";
+import "../Crowdsale.sol";
+import "../../ownership/Ownable.sol";
+
+
+/**
+ * @title UserCappedCrowdsale
+ * @dev Crowdsale with per-user caps.
+ */
+contract UserCappedCrowdsale is Crowdsale, Ownable {
```

Comment:
Maybe we can think of a better name?

Reply:
decided offline with @facuspagnuolo and @fiiiu to go with `IndividuallyCappedCrowdsale`